crates/environ/src/vmoffsets.rs

```diff
@@ -332,6 +338,23 @@ pub trait PtrSize {
         .unwrap()
     }
 
+    // Offsets within `VMContObj`
+
+    /// Return the offset of `VMContObj::contref`
+    fn vmcontobj_contref(&self) -> u8 {
+        0
+    }
+
+    /// Return the offset of `VMContObj::revision`
+    fn vmcontobj_revision(&self) -> u8 {
+        8
```

Comment:
To work across non-64-bit ISAs, which we support today via, for example, the 32-bit Pulley interpreter, this needs to be

```suggestion
        self.size()
```